src/lib.rs

```diff
@@ -831,7 +830,8 @@ impl ProgramVersion<'_> {
                     match map.definition.map_type {
                         bpf_map_type::BPF_MAP_TYPE_PERF_EVENT_ARRAY => {
                             if map.definition.max_entries == 0 {
-                                map.definition.max_entries = get_cpus()?.len() as u32;
+                                map.definition.max_entries =
+                                    cpu_info::max_possible_index()? as u32 + 1;
```

Comment:
@rc-vbundage we should move this out of the for-loop so we aren't recalculating it again and again. We should have caught this, sorry!